zebra-network/src/peer_set/initialize.rs

```diff
@@ -329,8 +344,10 @@ where
         warn!(
             ?initial_peers_len,
             ?peerset_initial_target_size,
-            "no successful initial peer connections"
+            "no successful initial peer connections, starting crawler anyway"
         );
+        // this redundant update will start the crawler
+        let _ = success_count_tx.send(success_count);
```

Comment:
Use the initial peer success count for the fanout limit, Part 1b

- Start the crawler anyway if all connections have been attempted

Reply:
Is this necessary? At the end of the method `success_count_tx` will get dropped, resuming the `changed()` future anyway, right?

Reply:
I think you're right, I wasn't really paying attention to the error.

Reply:
This code is removed in the latest #2160.